terraform/github-actions-deployment/01-variables.tf

```
@@ -0,0 +1,37 @@
+variable "region" {
+  type    = string
+  default = "eu-central-2"
+}
+
+variable "environment" {
+  type    = string
+  default = "dev"
+}
+
+variable "tfstate_bucket_name" {
+  type = string
+}
+
+variable "organization_name" {
+  type = string
+}
+
+variable "repo_name" {
+  type = string
+}
```

a description here would be super helpful, because it's hard to guess what these variables should actually contain. you may also want to consider renaming:

```suggestion
variable "tfstate_bucket_name" {
  type = string
}

variable "github_account_name" {
  type = string
}

variable "github_repo_name" {
  type = string
}
```